doc/Makefile

```
@@ -45,17 +35,15 @@ api:
 	@echo
 	$(SPHINXAUTOGEN) -i -t _templates -o api/generated api/*.rst
 
-
-linkcheck:
-	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
-	@echo
-	@echo "Link check complete; look for any errors in the above output " \
-	      "or in $(BUILDDIR)/linkcheck/output.txt."
-
-doctest:
-	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
-	@echo "Testing of doctests in the sources finished, look at the " \
-	      "results in $(BUILDDIR)/doctest/output.txt."
-
-serve:
+server:
 	cd $(BUILDDIR)/html && python -m http.server 8009
```

Add more messages so that people know how to preview the documentation.
```suggestion
server:
    @echo
    @echo "Running a server on port 8009."
    @echo "Open http://localhost:8009 in a web browser to preview the documentation."
    @echo
	cd $(BUILDDIR)/html && python -m http.server 8009
```